src/pieces/api/assets.py

```diff
@@ -156,56 +155,48 @@ def delete_asset_by_id(asset_id):
     except Exception as e:
         return f"Failed to delete {asset_id}"
 
-def update_asset(asset_id, application):
-    ## NOT CURRENTLY WORKING ##
 
+def reclassify_asset(asset_id, classification):
     asset_api = pos_client.AssetApi(api_client)
+    with open(extensions_dir) as f:
+        extension_mapping = json.load(f)
+        if classification not in extension_mapping:
+            show_error(f"Invalid classification: {classification}","Please choose from the following: \n "+", ".join(extension_mapping.keys()))
+            return
+        
+    try:
+        response = asset_api.asset_reclassify(asset_reclassification=
+                                              pos_client.AssetReclassification(ext=classification,asset=asset_api.asset_snapshot(asset_id)))
+        print(f"reclassify the asset to {classification} successfully")
+    except Exception as e:
+        show_error("Error reclassifying asset: ",{e})
 
-    working_asset = get_asset_by_id(asset_id)
-    # file_name = working_asset.get('name')
-    file_name = "Pieces_is_the_Best.tex" ################ TEST
-
-    # Define the path to the file
-    file_path = f"opened_snippets/{file_name}"
 
-    # Open and read the file content
+def update_asset(file_path,asset_id):
     try:
-        with open(file_path, 'r') as file:
-            new_content = file.read()
+        with open(file_path,"r") as f:
+            data = f.read()
     except FileNotFoundError:
-        print(f"File not found: {file_path}")
+        show_error("Error in update asset","File not found")
         return
-    except Exception as e:
-        show_error("Error reading file: ",{e})
+    asset_api = pos_client.AssetApi(api_client)
+    format_api = pos_client.FormatApi(api_client)
+
+    # get asset
+    created = asset_api.asset_snapshot(asset_id, transferables=False)
+
+    # update the original format's value
+    original = format_api.format_snapshot(created.original.id, transferable=True)
```

Comment:
Again, this flow is going to be different if it is an OCR snippet. Can be its own issue unless you want to layer it into this PR.